packages/screens/Organizations/components/MembershipOrg/MembershipMemberSettingsSection.tsx

```diff
@@ -21,13 +21,13 @@ import {
   ORGANIZATION_DEPLOYER_STEPS,
 } from "@/utils/types/organizations";
 
-interface MemberSettingsSectionProps {
+interface MembershipMemberSettingsSectionProps {
```

Comment:
i don't like the naming but i can't think of something that answer this:
- should describe the kind of org, here a membership based org
- should describe the section here : Member Settings